packages/components/atoms/f-card/src/components/Card.vue

```diff
@@ -2,24 +2,31 @@
     <div
         data-test-id="card-component"
         :class="[
-            $style['c-card'],
-            (isRounded ? $style['c-card--rounded'] : ''),
-            (hasOutline ? $style['c-card--outline'] : ''),
-            (isPageContentWrapper ? $style['c-card--pageContentWrapper'] : '')
-        ]"
-    >
-        <component
-            :is="cardHeadingTag"
-            v-if="cardHeading"
-            :class="[
-                $style['c-card-heading'],
-                (cardHeadingPosition !== 'left' ? $style[`c-card--${cardHeadingPosition}`] : '')
-            ]"
-            data-test-id="card-heading"
-        >
-            {{ cardHeading }}
-        </component>
-        <slot />
+            $style['c-card'], {
+                [$style['c-card--rounded']]: isRounded,
+                [$style['c-card--outline']]: hasOutline,
+                [$style['c-card--pageContentWrapper']]: isPageContentWrapper
+            }]">
+        <div :class="[$style['c-card--innerSpacing']]">
```

Comment:
This class would just be a child of `c-card` (not a modifier). So would just be called `c-card-innerSpacing`.

We've tended to use `wrapper` or `contentWrapper` in other components I think, so could also call it `c-card-contentWrapper` or something like that?

Reply:
I agree with `c-card-innerSpacing`. In terms of `c-card-contentWrapper` there is already `c-card--pageContentWrapper` class and it feels a bit confusing to understand the difference between them, as with `c-card-innerSpacing` it is more understandable straight away imo